dev-tools/repl-utils/src/lib.rs

```diff
@@ -148,12 +150,21 @@ fn process_entry<C: Parser>(
         return LoopResult::Continue;
     }
 
-    // Parse the line of input as a REPL command.
+    // Split on the first pipe (`|`) character if it exists. Use the first
```

Comment:
I swear I fixed this. Must have been on one of my other branches.

Reply:
fixed in [2a0559c](https://github.com/oxidecomputer/omicron/pull/8878/commits/2a0559c5684345a56724f79b1662b67b2d167442)